homeassistant/components/incomfort/config_flow.py

```diff
@@ -63,9 +78,60 @@ async def async_try_connect_gateway(
     return None
 
 
+class InComfortOptionsFlowHandler(OptionsFlow):
```

Comment:
Let's put this one below the ConfigFlow

Reply:
The config flow holds a reference to the options flow, so it is easier to define the OptionFlowHandler first

Reply:
It would require an import of:

```python
from .config_flow import InComfortOptionsFlowHandler
```

Not sure that would be wise to do.

Reply:
An import of

```python
from __future__ import annotations
```

did the trick. Moved the options flow.